src/scss/components/_filter.scss

```diff
@@ -0,0 +1,196 @@
+// Styles for the filter
+
+// CSS classes for "apply filter" and "reset filter" buttons
+.stroke-black {
+	stroke: var(--currentColor, black);
+}
+
+.fill-black {
+	fill: var(--currentColor, black);
+}
+
+.fill-c4c4c4 {
+	fill: var(--currentColor, #c4c4c4);
+}
+
+// CSS classes for the "expand filter" button
+.expand-svg {
+	fill: var(--expand-button-color, $gold-light);
+	stroke: var(--expand-button-stroke-color, $blue-alt-light);
+	stroke-width: rem(4);
+}
+
+.filter {
+	border: 1px solid $blue-alt-light;
+	margin-bottom: rem(48);
+
+	// Transparent buttons
+	button {
+		background-color: transparent;
+		background-repeat: no-repeat;
+		border: none;
+		cursor: pointer;
+		outline: none;
+		overflow: hidden;
+	}
+
+	// Custom checkboxes. Reference: https://codepen.io/spacemonkey/pen/vmZROv
+	.filter-checkbox {
+		opacity: 0; // hide it
+		position: absolute; // take it out of document flow
+
+		& + label {
+			cursor: pointer;
+			padding: 0;
+			position: relative;
+		}
+
+		// Box.
+		& + label::before {
+			border: 1px solid $blue-alt-light;
+			content: '';
+			display: inline-block;
+			height: rem(20);
+			margin-right: rem(10);
+			vertical-align: text-top;
+			width: rem(20);
+		}
+
+		// Box focus
+		&:focus + label::before,
+		&:hover + label::before {
+			border: 2.5px solid $blue-alt-light;
+		}
+
+		// Box checked
+		&:checked + label::before {
+			background: $blue-alt-light;
+		}
+
+		// Checkmark
+		&:checked + label::after {
+			background: $blue-alt-light;
+			box-shadow:
+				2px 0 0 white,
+				4px 0 0 white,
+				4px -2px 0 white,
+				4px -4px 0 white,
+				4px -6px 0 white,
+				4px -8px 0 white;
+			content: '';
+			height: 2px;
+			left: rem(5);
+			position: absolute;
+			top: rem(9);
+			transform: rotate(45deg);
+			width: 2px;
+		}
+	}
+	// End of custom checkboxes
+
+	.filter-header {
+		background-color: $gold-light;
+		display: flex;
+		flex-direction: row;
+		justify-content: space-between;
+
+		h2 {
+			border-bottom: none;
+			font-size: rem(24);
+			line-height: rem(8);
+			padding-left: rem(8);
+		}
+
+		button {
+			margin-right: rem(16);
+		}
+
+		svg {
+			height: rem(24);
+			width: rem(24);
+		}
+	}
+
+	.filter-choices {
+		display: grid;
+		grid-template-columns: repeat(1, auto);
+		list-style-type: none;
+		margin: 0;
+		padding: 0;
+
+		li {
+			padding: rem(4) rem(8);
+		}
+	}
+
+	.filter-buttons {
+		background-color: $grey-lighter;
+		border-top: 1px solid $blue-alt-light;
+		display: flex;
+		justify-content: flex-end;
+
+		svg {
+			fill: none;
+			height: rem(40);
+			margin-right: rem(3);
+			padding: rem(3);
+			width: rem(40);
+		}
+
+		span {
+			font-size: rem(12);
+		}
+	}
+
+	.apply-button,
+	.reset-button {
+		&:focus,
+		&:hover {
+			svg {
+				background-color: $gold-light;
+			}
+		}
+	}
+
+	.filter-expand-button:focus,
```

Comment:
For the Tags Filters svg arrow Icon. I believe the solution @uttaraghodke decided on at todays meeting was to have it filled when expanded and not filled other wise. A styling that is uses these selectors `button[aria-expanded="true"]` or `.filter-body[style="display: block;"]` would better accomplish this. 

Also using `stroke-width: 0.25;` inside the selector instead of changing the colour of the stroke would prevent the icon form looking smaller while filled and Tag Filters is expanded.

Reply:
> For the Tags Filters svg arrow Icon. I believe the solution @uttaraghodke decided on at todays meeting was to have it filled when expanded and not filled other wise. A styling that is uses these selectors `button[aria-expanded="true"]` or `.filter-body[style="display: block;"]` would better accomplish this.
> 
@uttaraghodke: could you clarify the filled expand button should be applied when the button is hovered/focused, or when the filter is expanded?

> Also using `stroke-width: 0.25;` inside the selector instead of changing the colour of the stroke would prevent the icon form looking smaller while filled and Tag Filters is expanded.
>
@TeddTech: do you mean `0.25rem`? This is an equivalent of `4px` which is already [the default stroke width](https://github.com/cindyli/wecount.inclusivedesign.ca/blob/feat/add-filter-to-views/src/scss/components/_filter.scss#L20) for the expand button. It is the cause of the uneven button top you mentioned in the previous comment. I believe our decision from yesterday was to remove the stroke width when the button is filled to fix the uneven button top.

Reply:
No `stroke-width: 0.25;` is not the same as `stroke-width: 0.25rem;`. By reducing `stroke-width` it can allow the fill to appear bigger.